pages/room/[room_id]/addfrivia.js

```diff
@@ -0,0 +1,166 @@
+              htmlFor="frivia"
+              className="text-text absolute -top-1.5 w-full h-full pointer-events-none flex "
+            >
+              Question here
+            </label>
+          
+            <h2 className="text-text">
+              Give the user 4 possible answers and mark the right one
+            </h2>
+            <div>
+            <div className="m-1 flex items-center mb-4">
+              <input
+                autoComplete="off"
+                tpye="text"
+                id="answer1"
+                name="answer1"
+                className="bg-transparent border border-1 border-gray-300 text-center sadarounded-lg text-text w-screen mr-1"
+              />
+              
+              <input type="radio" name="answerCorrect" value="answer1" className='h-5 w-5'/>
+              
+
+              <label
+                htmlFor="answer1"
+                className="text-text absolute -top-1.5 w-full h-full pointer-events-none flex"
+              ></label>
+            </div>
+            <div className="m-1 flex items-center mb-4">
+              <input
+                autoComplete="off"
+                type="text"
+                id="answer2"
+                name="answer2"
+                className="bg-transparent border border-1 border-gray-300 text-center sadarounded-lg text-text w-screen mr-1"
+              />
+              <input type="radio" name="answerCorrect" value="answer2" className='h-5 w-5' />
+              <label
+                htmlFor="answer2"
+                className="text-text absolute -top-1.5 w-full h-full pointer-events-none flex "
+              ></label>
+            </div>
+            <div className="m-1 flex items-center mb-4">
+              <input
+                autoComplete="off"
+                type="text"
+                id="answer3"
+                name="answer3"
+                className="bg-transparent border border-1 border-gray-300 text-center sadarounded-lg text-text w-screen mr-1"
+              />
+              <input type="radio" name="answerCorrect" value="answer3" className='h-5 w-5'/>
+              <label
+                htmlFor="answer3"
+                className="text-text absolute -top-1.5 w-full h-full pointer-events-none flex "
+              ></label>
+            </div>
+            <div className="m-1 flex items-center mb-4">
+              <input
+                autoComplete="off"
+                type="text"
+                id="answer4"
+                name="answer4"
+                className="bg-transparent border border-1 border-gray-300 text-center sadarounded-lg text-text w-screen mr-1"
+              />
+              <input type="radio" name="answerCorrect" value="answer4" className='h-5 w-5' />
+              <label
+                htmlFor="answer4"
+                className="text-text absolute -top-1.5 w-full h-full pointer-events-none flex "
+              ></label>
+            </div>
+            </div>
```

Comment:
I would suggest working with components here.

the 4 input fields are almost identical and the only difference is their value attribute
create one component that contains 
<div>
      <input />
      <input />
      <label > </label>
</div>
and display that component 4 times. you can hand down the value prop in the respective parent element